velox/exec/MergeJoin.h

```diff
@@ -225,7 +225,7 @@ class MergeJoin : public Operator {
   /// Evaluates join filter on 'filterInput_' and returns 'output' that contains
   /// a subset of rows on which the filter passed. Returns nullptr if no rows
   /// passed the filter.
-  RowVectorPtr applyFilter(const RowVectorPtr& output);
+  RowVectorPtr applyFilter(RowVectorPtr& output);
```

Comment:
why isn't is const anymore? I thought the idea is that applyFilter() always returns a dictionary wrapped around it?

Reply:
@pedroerp For the full outer join, we may need to assign the `fullOuterOutput_ ` to `output`.